### PR TITLE
add yire morlans linkfree page

### PR DESCRIPTION
**Pull request by yiremorlans**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Thanks for this open source linktree alternative!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1665"></a>

### CODE REVIEW

File: public/data/yiremorlans.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      "url": "https://twitter.com/xenaaMorph",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/yiremorlans/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Join the 100devs community on discord",
```
Comment (Panquesito7) [inline public/data/yiremorlans.json:28]: I recommend capitalizing the `d` here. What do you think?
```suggestion
      "name": "Join the 100devs community on Discord",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Comment (yiremorlans) [reply public/data/yiremorlans.json:28]: Excellent! Thanks for catching that!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! Thanks for contributing! 🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Panquesito7 — APPROVED** [review]
Nice! Thank you. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by maharshi-sinha — COMMENTED** [review]
👍

**Pull request merged**